website/src/components/RootNavbar/index.tsx

```diff
@@ -22,12 +22,6 @@ export default function RootNavbar() {
         document.getElementById("product-dropdown-link")
       );
     }
-    if (!docsDropdown) {
-      docsDropdown = new Dropdown(
```

Comment:
Removed the link to legacy docs from the Navbar.

Reply:
Nothing blocking here, just wondering if we provide a link anywhere to the legacy docs?

Reply:
Not directly from the navbar, but we do in relevant places in the new docs, and they're still being crawled by Algolia and Google.

Just was thinking most people clicking that link now will be 1.0 users and didn't want to confuse them.

Reply:
Yeah, totally makes sense.  Was just personally curious if there was anywhere left on the site that did have a link.